diagrams/__init__.py

```diff
@@ -487,7 +490,8 @@ def append(self, other: Union[List["Node"], List["Edge"]], forward=None, reverse
                 self._attrs = o.attrs.copy()
                 result.append(o)
             else:
-                result.append(Edge(o, forward=forward, reverse=reverse, **self._attrs))
+                result.append(
```

Comment:
Why is this line changed?

Reply:
Auto formatter must have done that. It doesn't improve readability much though, I will revert it.